R/tune_sim_anneal.R

```diff
@@ -354,7 +358,7 @@ tune_sim_anneal_workflow <-
 
     cols <- tune::get_tune_colors()
     if (control$verbose_iter) {
-      rlang::inform(cols$message$info(paste("Optimizing", metrics_name)))
```

Comment:
This is a lot to read though: 

```
→ 1 | warning: Some 'x' values beyond boundary knots may cause ill-conditioned basis
functions.
There were issues with some computations   1: x2
Optimizing roc_auc                               
Initial best: 0.88440                            
1 ◯ accept suboptimal  roc_auc=0.88335 (+/-0.01352)
2 + better suboptimal  roc_auc=0.88418 (+/-0.01354)
3 ◯ accept suboptimal  roc_auc=0.88374 (+/-0.01421)
4 ♥ new best  roc_auc=0.88592 (+/-0.01306)        
→ 2 | warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
5 ◯ accept suboptimal  roc_auc=0.88444 (+/-0.01215)       
6 ◯ accept suboptimal  roc_auc=0.88102 (+/-0.01224)       
7 + better suboptimal  roc_auc=0.88473 (+/-0.01238)       
8 ◯ accept suboptimal  roc_auc=0.88385 (+/-0.01252)       
9 ◯ accept suboptimal  roc_auc=0.8836 (+/-0.0132)         
10 + better suboptimal  roc_auc=0.88505 (+/-0.01277)      
There were issues with some computations   1: x22   2: x4
```

The mixing of numbers for iterations and warnings/errors is confusing. 

Would we consider using uppercase letters for the error/warning logging? We could have the pool of labels are `c(LETTERS, letters)` if we think that there will be more than 52 unique cases.

Reply:
On board! PR at tidymodels/tune#596. :)